evalml/tests/pipeline_tests/test_pipeline_utils.py

```diff
@@ -212,74 +227,19 @@ def test_make_pipeline_no_nulls(input_type, problem_type):
                     + delayed_features
                     + estimator_components
                 )
-            pipeline.component_graph.compute_order == [
+            assert pipeline.component_graph.compute_order == [
                 component.name for component in expected_components
             ]
 
 
 @pytest.mark.parametrize("input_type", ["pd", "ww"])
 @pytest.mark.parametrize("problem_type", ProblemTypes.all_problem_types)
-def test_make_pipeline_no_datetimes(input_type, problem_type):
```

Comment:
I didn't see value in this compared to `test_make_pipeline_text_columns` so deleting :)